codecs/webp/Makefile

```diff
@@ -1,19 +1,25 @@
-CODEC_URL := https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.1.0.tar.gz
+CODEC_URL = https://github.com/RReverser/libwebp/archive/emscripten-simd.tar.gz
```

Comment:
Can you add a comment explaining when we can switch back to the upstream repo? Ideally with a link to the issue/PR or something that needs to land.